services/core-web/src/components/Forms/Securities/BondForm.js

```diff
@@ -29,12 +29,14 @@ const propTypes = {
   closeModal: PropTypes.func.isRequired,
   title: PropTypes.string.isRequired,
   submitting: PropTypes.bool.isRequired,
+  pristine: PropTypes.bool.isRequired,
```

Comment:
I think we've used the flag is_dirty or 'dirty' to indicate if an object has been changed, can you take a look? If so, we should keep this consistent. If not, pristine is fine.

Reply:
Pristine is the opposite of Dirty, its redundant to have two but that's what Redux Forms does. I've noticed we've used Pristine throughout the app already and don't use Dirty.

Reply:
sounds good